src/components/tip-group/tip-group.scss

```diff
@@ -8,12 +8,5 @@
 
 ::slotted(calcite-tip) {
   @apply m-0 border-none;
-}
-
-:host-context(calcite-tip-manager) {
-  @apply my-3;
-}
-
-:host-context(calcite-tip-manager) ::slotted(calcite-tip) {
```

Comment:
Can we setup a lint error/warning for using host-context so its not checked into any files?

Reply:
Good idea. I'll see what we can set up for this.